docs/1_usage.md

```diff
@@ -2,69 +2,53 @@
 
 Lean how to use the ReVanced CLI.
 
-## ⚡ Setup (optional)
+## ⚡ Setup
 
-1. Make sure your device is connected
+1. Download Java 19: https://jdk.java.net/19/ then unzip it
+2. Download latest release (.jar) of revanced CLI: https://github.com/revanced/revanced-cli/releases
+3. Download latest release (.jar) of revanced patches: https://github.com/revanced/revanced-patches/releases
+4. Download latest release (.apk) of revanced integrations: https://github.com/revanced/revanced-integrations/releases
+5. Download the appropriate version of the Youtube APK: https://www.apkmirror.com/apk/google-inc/youtube/youtube-18-03-36-release/ (replace the version in the URL if necessary; make sure to download the APK and not the APK bundle)
 
-   ```bash
-   adb shell exit
-   ```
-
-   If you plan to use the root variant, check if you have root access
-
-   ```bash
-   adb shell su -c exit
-   ```
-
-2. Copy the ADB device name
-
-   ```bash
-   adb devices
-   ```
+Put all of that in the same directory for easier usage.
 
 ## 🔨 ReVanced CLI Usage
 
+You may need to slightly modify the commands to adapt to your specific versions of Java and/or Revanced.
```

Comment:
This wouldn't be necessary if you ran `./java`. But now the usage is constrained specifically to OpenJDK, a version and the ReVanced CLI version instead of being universally correct. This should not be the case.